src/toxcore_tests/dht_tests.rs

```diff
@@ -1340,6 +1493,18 @@ fn nat_ping_from_bytes_test() {
     with_bytes(ping);
 }
 
+// NatPing::parse_bytes()
+
+#[test]
+fn nat_ping_parse_bytes_rest_test() {
+    fn with_bytes(bytes: Vec<u8>) {
+        if let Ok(Parsed(_, rest)) = NatPing::parse_bytes(&bytes) {
```

Comment:
wouldn't it be better to use `NatPing` as function arg, `.to_bytes()` it append some random bytes & test that?